indigo_api/templatetags/indigo.py

```diff
@@ -19,3 +19,49 @@ def work_resolver_url(context, work):
         frbr_uri = '/akn' + frbr_uri
 
     return context.get('resolver_url', settings.RESOLVER_URL) + frbr_uri
+
+
+@register.simple_tag
+def commenced_provisions_description(document, commencement, uncommenced=False):
+    work = document.work
+    if uncommenced:
+        uncommenced_provisions = document.work.uncommenced_provisions()
+        # TODO: same as below for uncommenced provisions
+        return ', '.join([p.id for p in uncommenced_provisions])
+
+    commenceable_provisions = work.commenceable_provisions()
+    commenced_provisions = commencement.provisions
+    runs = []
+    current_run = []
+
+    def end_current_run(run):
+        if len(run) < 4:
+            run = [f'{item["type"]} {item["num"]}' for item in run]
+        else:
+            run = [f'{run[0]["type"]}s {run[0]["num"]} to {run[-1]["num"]}']
```

Comment:
Needs to be localised 😱

Reply:
Actually, let's make our lives easier: no plural on the type (eg. "Section 1, 2, 3 to 5"), and then translate "to" entirely on its own.

Reply:
how do you mean 'translate "to" entirely on its own'? I like the no plural though!

Reply:
Translate "to" and then include it in the string, rather than translating the whole string "X to Y"